pkg/sfu/streamallocator.go

```diff
@@ -142,10 +145,12 @@ type StreamAllocator struct {
 	abortedProbeClusterId ProbeClusterId
 	probeTrendObserved    bool
 	probeEndTime          time.Time
-	probeEstimator        *Estimator
+	probeChannelObserver  *ChannelObserver
 
 	prober *Prober
 
+	channelObserver *ChannelObserver
```

Comment:
The `Estimator` from the previous version is replaced by `ChannelObserver` which combines both the channel estimate + NACK based monitoring and produces the combined trend. The new struct is towards the end of this file